plugin/federation/federation_test.go

```diff
@@ -10,12 +10,13 @@ import (
 )
 
 func TestInjectSources(t *testing.T) {
-	var cfg config.Config
-	f := &federation{}
-	f.InjectSources(&cfg)
-	if len(cfg.AdditionalSources) != 2 {
-		t.Fatalf("expected an additional source but got %v", len(cfg.AdditionalSources))
-	}
+	cfg, err := config.LoadConfig("test_data/gqlgen.yml")
+	require.NoError(t, err)
+ 	f := &federation{}
+ 	f.InjectSources(cfg)
+ 	if len(cfg.AdditionalSources) != 2 {
+ 		t.Fatalf("expected 2 additional sources but got %v", len(cfg.AdditionalSources))
```

Comment:
Not related to your diff, but it seems kinda confusing that some assertions are done through `require` and some though `t.` functions, but maybe there's a good reason for htat?

Reply:
Yeah, I tried to change them all to use require and the tests started failing.  I didn't see the need to try to figure out why...